packages/Sandblocks-Core/SBEditor.class.st

```diff
@@ -434,17 +444,13 @@ SBEditor >> countModifier [
 
 { #category : #'submorphs-add/remove' }
 SBEditor >> createArtefactInView: aMorph [
-	" open this morph and mark it has currently being unsaved "
+	 " open this morph and mark it has currently being unsaved "
 
-	self assert: aMorph isArtefact.
-	
-	self do: (SBEditorOpenMorphCommand new
-		morph: aMorph;
-		editor: self sandblockEditor;
-		isUnsaved: true;
-		yourself).
-	
-	^ aMorph
+    self assert: aMorph isArtefact.
+
+    self do: (self openMorphInViewCommand: aMorph).
+
+    ^ aMorph
```

Comment:
This opens the artefact in view (instead of being somewhere far off) but then it no longer has the isUnsaved property, which the insertCommand doesn't support yet.